active-java-processes/check_active_java_processes.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+number_of_java_processes=0
+
+#get number of runing java processes except sun.tools.jps.Jps -Dapplication.home=/usr/lib/jvm/....
+number_of_java_processes=$(jps -vl | grep -v 'sun.tools.jps.Jps' | grep -v 'hudson.remoting.jnlp.Main' | grep -v grep | wc -l)
+java_processes=$(jps -vl | grep -v 'sun.tools.jps.Jps' | grep -v 'hudson.remoting.jnlp.Main' | grep -v grep)
```

Comment:
We do not need to re-run the "jps -vl | grep -v 'sun.tools.jps.Jps' | grep -v 'hudson.remoting.jnlp.Main'  | grep -v grep" command twice. Please re-use.